code/game/objects/items/stacks/medical_packs.dm

```diff
@@ -125,6 +126,7 @@
 	heal_brute = 10
 	stop_bleeding = 1800
 	dynamic_icon_state = TRUE
+	parent_stack = TRUE
```

Comment:
no longer necessary to define this here since it's TRUE by inheritance

Reply:
fixed in latest push

Reply:
It should default to FALSE, as this is the item for the normal trauma pack, its marked true to have its stacking handled separately then bruise_pack/advanced or /improvised